ROMFS/px4fmu_common/mixers/standard_vtol_hitl.main.mix

```diff
@@ -0,0 +1,26 @@
+Mixer for standard vtol plane (SITL) with motor x configuration
+=========================================================
+
+This file defines a single mixer for a standard vtol plane (SITL gazebo) with motors in X configuration. The plane has two ailerons and one elevator.
```

Comment:
Everywhere:
- VTOL not vtol 
- This config is referred to as a quadplane in our docs. 
- do you mean SITL or HITL? 

So possibly you mean:
```
Mixer for Standard VTOL (QuadPlane) with motor x configuration (Gazebo HITL)
=========================================================

This file defines a single mixer for a Standard VTOL QuadPlane (HITL Gazebo) with motors in X configuration. The plane has two ailerons and one elevator.
```